26_leet_removeduplicates.py

```diff
@@ -0,0 +1,15 @@
+from typing import List
+
+
+class Solution:
+    def removeDuplicates(self, nums: List[int]) -> int:
+        i: int = 1  # initialize nums tracker
```

Comment:
No need to specify type when it can be automatically derived from assignment.

For example `x = {}` can't be derived, because we don't know what kind of dict that might be, is it `Dict[str, str]`, or maybe `Dict[float, Dict[int, Tuple[int, int]]]`?

However with `x = 1`  the type of x is derived automatically, since no way it can be anything other than `int`